Sources/Helpers/OASettingsHelper.mm

```diff
@@ -520,8 +527,31 @@ - (BOOL) readFromFile:(NSString *)filePath error:(NSError * _Nullable *)error
     [self.item applyRoutingPreferences:routingSettings];
     
     [OsmAndApp.instance.data.mapLayerChangeObservable notifyEvent];
+}
+
+- (void) saveToBackup:(NSDictionary<NSString *, NSString *> *)settings withFilename:(NSString *)filename
```

Comment:
It is not clear why applyReadedSettings, saveToBackup and restoreFromBackup added here. Looks like wrong approach.